public/index.html

```diff
@@ -11,10 +11,24 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/faviconOktaTools.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-
+    <meta
+      name="description"
+      content="Okta Tools is one trusted platform to secure every identity, from customers to your workforce with Single Sign-On, Multi-factor Authentication, Lifecycle Management, and more."
+    />
+    <meta property="og:site_name" content="Okta Tools" />
+    <meta property="og:title" content="Okta | Tools" />
+    <meta property="og:url" content="https://oktatools.tecnics.com" />
+    <meta
+      property="og:description"
+      content="Okta Tools is one trusted platform to secure every identity, from customers to your workforce with Single Sign-On, Multi-factor Authentication, Lifecycle Management, and more."
```

Comment:
@niyatip25 , I think the description could be 
"Okta tools, parse, inspect, decode, test SAML and other Okta identity features."

Reply:
@MurthyAvanithsa 
Added this "Okta tools allows to parse, inspect, decode, test SAML and other Okta identity features."